### PR TITLE
r/storagegateway_stored_iscsi_volume - add new resource

### PR DESCRIPTION
**Pull request by DrFaust92**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #10723

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
resource_aws_storagegateway_stored_iscsi_volume: add new resource
```

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccAWSStorageGatewayStoredIscsiVolume_'
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_basic (499.02s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_kms (481.74s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_tags (648.69s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId (515.79s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_disappears (708.12s)
```

### CODE REVIEW

File: aws/provider.go [hunk 1]
```diff
@@ -868,6 +868,7 @@ func Provider() *schema.Provider {
 			"aws_storagegateway_nfs_file_share":                       resourceAwsStorageGatewayNfsFileShare(),
 			"aws_storagegateway_smb_file_share":                       resourceAwsStorageGatewaySmbFileShare(),
 			"aws_storagegateway_upload_buffer":                        resourceAwsStorageGatewayUploadBuffer(),
+			"aws_storagegateway_stored_iscsi_volume":                  resourceAwsStorageGatewayStoredIscsiVolume(),
```
Comment (ewbankkit) [inline aws/provider.go:871]: Nit. Move up 1 to make alphabetical.

Comment (DrFaust92) [reply aws/provider.go:870]: Thanks, changed

File: aws/resource_aws_storagegateway_stored_iscsi_volume.go [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+			},
+			"lun_number": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"target_arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"volume_arn": {
```
Comment (ewbankkit) [inline aws/resource_aws_storagegateway_stored_iscsi_volume.go:90]: Do we need both `arn` and `volume_arn`? They are always set to the same value.

**Comment by ewbankkit** [comment]
```console
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSStorageGatewayStoredIscsiVolume_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSStorageGatewayStoredIscsiVolume_ -timeout 120m
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
    resource_aws_storagegateway_stored_iscsi_volume_test.go:62: Check 2/3 error: aws_storagegateway_stored_iscsi_volume.test: Attribute 'kms_encrypted' expected "true", got "false"
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
    resource_aws_storagegateway_stored_iscsi_volume_test.go:173: [INFO] Got non-empty plan, as expected
--- FAIL: TestAccAWSStorageGatewayStoredIscsiVolume_kms (382.85s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId (415.53s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_basic (427.30s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_tags (471.82s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_disappears (490.76s)
FAIL
FAIL	github.com/terraform-providers/terraform-provider-aws/aws	490.804s
FAIL
GNUmakefile:27: recipe for target 'testacc' failed
make: *** [testacc] Error 1
```

File: aws/resource_aws_storagegateway_stored_iscsi_volume.go (hunk 2 above)
Comment (DrFaust92) [reply aws/resource_aws_storagegateway_stored_iscsi_volume.go]: good point, only for api parity but ill remove it and in can be added later if needed

**Comment by DrFaust92** [comment]
@ewbankkit, weird. just reran test and I'm not getting the error you are getting.

File: website/docs/r/storagegateway_stored_iscsi_volume.html.markdown [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+In addition to all arguments above, the following attributes are exported:
+
+* `arn` - Volume Amazon Resource Name (ARN), e.g. `arn:aws:storagegateway:us-east-1:123456789012:gateway/sgw-12345678/volume/vol-12345678`.
+* `chap_enabled` - Whether mutual CHAP is enabled for the iSCSI target.
+* `id` - Volume Amazon Resource Name (ARN), e.g. `arn:aws:storagegateway:us-east-1:123456789012:gateway/sgw-12345678/volume/vol-12345678`.
+* `lun_number` - Logical disk number.
+* `network_interface_port` - The port used to communicate with iSCSI targets.
+* `target_arn` - Target Amazon Resource Name (ARN), e.g. `arn:aws:storagegateway:us-east-1:123456789012:gateway/sgw-12345678/target/iqn.1997-05.com.amazon:TargetName`.
+* `volume_arn` - Volume Amazon Resource Name (ARN), e.g. `arn:aws:storagegateway:us-east-1:123456789012:gateway/sgw-12345678/volume/vol-12345678`.
```
Comment (ewbankkit) [inline website/docs/r/storagegateway_stored_iscsi_volume.html.markdown:64]: Need to remove `volume_arn` from the documentation also.

**Comment by ewbankkit** [comment]
No errors after pulling latest code:

```console
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSStorageGatewayStoredIscsiVolume_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSStorageGatewayStoredIscsiVolume_ -timeout 120m
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
    resource_aws_storagegateway_stored_iscsi_volume_test.go:171: [INFO] Got non-empty plan, as expected
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_basic (447.26s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_kms (449.97s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId (467.86s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_tags (501.61s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_disappears (553.77s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	553.825s
```

File: website/docs/r/storagegateway_stored_iscsi_volume.html.markdown (hunk 3 above)
Comment (DrFaust92) [reply website/docs/r/storagegateway_stored_iscsi_volume.html.markdown]: done.

**Review by ewbankkit — APPROVED** [review]
LGTM.

**Comment by breathingdust** [comment]
@DrFaust92 getting a consistent failure from the `TestAccAWSStorageGatewayStoredIscsiVolume_kms` test. Tried a couple accounts to be sure.

```console
make testacc TEST=./aws TESTARGS='-run=TestAccAWSStorageGatewayStoredIscsiVolume_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSStorageGatewayStoredIscsiVolume_ -timeout 120m
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
    resource_aws_storagegateway_stored_iscsi_volume_test.go:61: Check 2/3 error: aws_storagegateway_stored_iscsi_volume.test: Attribute 'kms_encrypted' expected "true", got "false"
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
    resource_aws_storagegateway_stored_iscsi_volume_test.go:171: [INFO] Got non-empty plan, as expected
--- FAIL: TestAccAWSStorageGatewayStoredIscsiVolume_kms (379.83s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_basic (387.48s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId (388.43s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_tags (450.49s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_disappears (488.49s)
FAIL
FAIL	github.com/terraform-providers/terraform-provider-aws/aws	489.846s
FAIL
make: *** [testacc] Error 1
```

**Comment by DrFaust92** [comment]
Looking into this, i think i need to wait for a volume status here.

**Comment by DrFaust92** [comment]
Added waiter and retested. still works for me but maybe there is some eventual consistency issue? i hope the waiter solves this.

**Review by ewbankkit — APPROVED** [review]
LGTM.
Running the full set of relevant acceptance tests showed a failure:

```console
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSStorageGatewayStoredIscsiVolume_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSStorageGatewayStoredIscsiVolume_ -timeout 120m
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
    resource_aws_storagegateway_stored_iscsi_volume_test.go:61: Check 2/3 error: aws_storagegateway_stored_iscsi_volume.test: Attribute 'kms_encrypted' expected "true", got "false"
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
    resource_aws_storagegateway_stored_iscsi_volume_test.go:171: [INFO] Got non-empty plan, as expected
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_basic (405.68s)
--- FAIL: TestAccAWSStorageGatewayStoredIscsiVolume_kms (408.92s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId (418.08s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_disappears (438.73s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_tags (522.47s)
FAIL
FAIL	github.com/terraform-providers/terraform-provider-aws/aws	522.513s
FAIL
GNUmakefile:26: recipe for target 'testacc' failed
make: *** [testacc] Error 1
```

but running the failing test in isolation succeeded:

```console
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSStorageGatewayStoredIscsiVolume_kms'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSStorageGatewayStoredIscsiVolume_kms -timeout 120m
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_kms (433.55s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	433.588s
```

**Comment by ewbankkit** [comment]
@DrFaust92 As both @breathingdust and I see the same error with the `kms_encrypted` value when running the full set of tests it may be worth adding a `waiter` on creation to wait for the volume status to hit `Available`: https://docs.amazonaws.cn/en_us/storagegateway/latest/userguide/managing-volumes.html#StorageVolumeStatuses.

**Comment by DrFaust92** [comment]
I was sure I added a waiter here. probably in branch with my old laptop /:. on it.

**Comment by DrFaust92** [comment]
Added waiter:

```
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_basic (515.62s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_disappears (555.03s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_tags (693.74s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_kms (536.17s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId (670.60s)
```

this means that the cached Iscsi volume resource also needs this waiter implemented.

File: aws/internal/service/storagegateway/waiter/waiter.go [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+const (
+	StoredIscsiVolumeAvailableTimeout = 5 * time.Minute
+)
+
+// StoredIscsiVolumeAvailable waits for a StoredIscsiVolume to return Available
+func StoredIscsiVolumeAvailable(conn *storagegateway.StorageGateway, volumeARN string) (*storagegateway.DescribeStorediSCSIVolumesOutput, error) {
+	stateConf := &resource.StateChangeConf{
+		Pending: []string{"BOOTSTRAPPING", "CREATING", "RESTORING"},
+		Target:  []string{"AVAILABLE"},
```
Comment (ewbankkit) [inline aws/internal/service/storagegateway/waiter/waiter.go:18]: Define these as constants in `aws/internal/service/storagegateway/waiter/status.go`?

**Review by ewbankkit — APPROVED** [review]
LGTM.

```console
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSStorageGatewayStoredIscsiVolume_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSStorageGatewayStoredIscsiVolume_ -timeout 120m
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_disappears (425.41s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_kms (453.96s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_tags (497.94s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_basic (501.85s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId (564.42s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	564.494s
```

**Comment by breathingdust** [comment]
LGTM :rocket: Thanks @DrFaust92!

Verified Acceptance Tests in Commercial (us-west-2)

```console
make testacc TEST=./aws TESTARGS='-run=TestAccAWSStorageGatewayStoredIscsiVolume_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSStorageGatewayStoredIscsiVolume_ -timeout 120m
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_tags
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_kms (410.86s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_disappears (424.94s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_basic (438.22s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_tags (552.31s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId (583.72s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	585.947s
```

Verified Acceptance Tests in GovCloud (us-gov-west-1)

```console
make testacc TEST=./aws TESTARGS='-run=TestAccAWSStorageGatewayStoredIscsiVolume_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSStorageGatewayStoredIscsiVolume_ -timeout 120m
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_kms
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== RUN   TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== PAUSE TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_basic
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_disappears
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_tags
=== CONT  TestAccAWSStorageGatewayStoredIscsiVolume_kms
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_basic (409.64s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_kms (419.51s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_tags (475.76s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_disappears (485.84s)
--- PASS: TestAccAWSStorageGatewayStoredIscsiVolume_snapshotId (558.56s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	560.409s
```

**Pull request merged**